pkgs/development/compilers/llvm/common/libunwind/default.nix

```diff
@@ -12,6 +12,7 @@
   python3,
   libcxx,
   enableShared ? !stdenv.hostPlatform.isStatic,
+  doFakeLibgcc ? stdenv.hostPlatform.useLLVM,
```

Comment:
This should also depend on `enableShared` I think? otherwise this produces dangling symlinks when building it as a static library.